### ISSUE TITLE
Unstable and freeze since today [Linux Ubuntu (Unity) on Nvidia]

### ISSUE THREAD

**Issue by Tyrben** [ValveSoftware/steam-for-linux#5305]
#### Your system information

* Steam client version (build number or date): 27/12/2017
* Distribution (e.g. Ubuntu): Ubuntu 64 16.04.2 LTS
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes
Card : Nvidia GeForce GTX 750 Ti
Driver : Nvidia 381.22

#### Please describe your issue in as much detail as possible:
Steam launches with glitches (or without). But everytime affect the whole Unity window manager : freeze for 5 seconds, every 10 seconds.
When starting with glitches, Unity parts have also glitches.

#### Steps for reproducing this issue:

1. Let 27/12/2017 minor update
2. Launch Steam

**Comment by LolmanCZ** [comment]
I think I have the same problem. Please someone help us.

**Comment by kisak-valve** [comment]
Hello @Tyrben, are you still experiencing this issue on an up to date system?

**Comment by Tyrben** [comment]
> 
> 
> Hello @Tyrben, are you still experiencing this issue on an up to date system?

Nope, but it came a time when I fully reinstalled my system and changed from Ubuntu to Debian. No more bug since then.
